java/core/src/main/java/com/whylogs/core/metrics/StandardMetric.java

```diff
@@ -0,0 +1,52 @@
+package com.whylogs.core.metrics;
+
+public enum StandardMetric {
+  /*
```

Comment:
Remove comment

Reply:
This are here so we know what types we need to come back to implement in the next pass (next week)